docs/installer_generation.md

```diff
@@ -2,26 +2,18 @@
 
 ODK Aggregate uses BitRock InstallBuilder to generate installer apps for Windows, Linux and Mac hosts.
 
-The InstallBuilder project files are located under `bitrock-installer` directory.
+The InstallBuilder project files are located under `installer` directory.
 
 ## Requirements
 
 **BitRock InstallBuilder**
 
 You can download it from [this website](https://installbuilder.bitrock.com/).
 
-**Maven**
-
-At the moment, the project leverages Maven to prepare it for the InstallBuilder application.
-
-**Notice:** This will be replaced by gradle tasks on the main Aggregate project
-
 ## Instructions
 
-1. Delete any WAR file present at `bitrock-installer` directory
-2. Generate a WAR file without any configuration files in it using `gradle clean build -x test -Pwar_mode=installer` command
```

Comment:
You seem to prefer `_` to `-`. I really like hyphens because they are easier to type. Your decision whether you keep using `_`.